### PR TITLE
Add arrayFromRange, set up test suite

### PR DESCRIPTION
**Pull request by jshakes**
- Adds an arrayFromRange function (would also be open to just calling this `range`) and a few tests
- Sets up Jest test suite; run with `npm test`
- Specify file types to be included when this package is installed via npm - i.e. only `.js` files and not tests, Readmes, etc.

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	presets: [
+		[
+			'@babel/preset-env',
```
Comment (chriszarate) [inline babel.config.js]: do we need babel? we aren't building and it doesn't seem like we're using new lang features or syntax (node 12 can handle a lot). if it needs to be transpiled for an older environment, that can be the job of the consuming codebase

**Review by chriszarate — APPROVED** [review]
one comment that can be addressed separately, otherwise :+1:

File: babel.config.js (hunk 1 above)
Comment (jshakes) [reply babel.config.js]: we only need it if we want to support `import` and `export` syntax, which is only supported in Node >13. It's probably a stretch to force people to use that so I'll just switch to using `require` instead

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	transform: {
+		'.*': 'babel-jest',
+	},
```
Comment (chriszarate) [inline jest.config.js]: remove?
```suggestion
```

**Pull request merged**